tsl/src/nodes/skip_scan/planner.c

```diff
@@ -593,15 +624,15 @@ sort_indexquals(IndexOptInfo *indexinfo, List *quals)
 }
 
 static OpExpr *
-fix_indexqual(IndexOptInfo *index, RestrictInfo *rinfo, AttrNumber distinct_column)
+fix_indexqual(IndexOptInfo *index, RestrictInfo *rinfo, AttrNumber table_attno)
 {
 	/* technically our placeholder col > NULL is unsatisfiable, and in some instances
 	 * the planner will realize this and use is as an excuse to remove other quals.
 	 * in order to prevent this, we prepare this qual ourselves.
 	 */
 
 	/* fix_indexqual_references */
-	int indexcol = get_idx_key(index, distinct_column);
+	int indexcol = get_idx_key(index, table_attno);
```

Comment:
could pass the index_attno directly instead of lookup here. Don't  see table_attno being used.